### PR TITLE
Release

### PR DESCRIPTION
**Pull request by ashirazist**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -9,7 +9,7 @@ Authors@R: c(
            person("Ruixue", "Wang", email = "ruixue.wang@merck.com", role = c("ctb")),
```
Comment (elong0527) [inline DESCRIPTION:9]: update my role to "aut", "cre"; remove Keaven's "cre" role. The current way will result in a note. Please refer gsDesign2.

File: .github/workflows/R-CMD-check.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
```
Comment (elong0527) [inline .github/workflows/R-CMD-check.yaml:1]: remove this one. check-standard is sufficient.

File: README.md [hunk 3]
```diff
@@ -1,28 +1,23 @@
----
-simtrial: Clinical Trial Simulation for Time-to-Event Endpoints
-output: github_document
----
+<!-- badges: start -->
+[![R build status](https://github.com/Merck/simtrial/workflows/R-CMD-check/badge.svg)](https://github.com/Merck/simtrial/actions)
```
Comment (elong0527) [inline README.md:2]: update badges

**Review by elong0527 — APPROVED** [review]
Could we merge this pull request to enable pkgdown and code coverage? The comments can be addressed in separate pull request.

File: .github/workflows/R-CMD-check.yaml (hunk 2 above)
Comment (ashirazist) [reply .github/workflows/R-CMD-check.yaml:1]: Done

**Comment by ashirazist** [comment]
> Could we merge this pull request to enable pkgdown and code coverage? The comments can be addressed in separate pull request.

We can do that. Except vignettes errors for pMaxCombo, the rest should be fine.

**Review by elong0527 — APPROVED** [review]

**Pull request merged**